x-pack/plugins/ml/server/routes/schemas/modules.ts

```diff
@@ -71,19 +71,11 @@ export const setupModuleBodySchema = schema.object({
   estimateModelMemory: schema.maybe(schema.boolean()),
 });
 
-export const getModuleIdParamSchema = (optional = false) => {
-  const stringType = schema.string();
-  return schema.object({
-    /**
-     * ID of the module.
-     */
-    moduleId: optional ? schema.maybe(stringType) : stringType,
-  });
-};
-
-export const optionalModuleIdParamSchema = getModuleIdParamSchema(true);
+export const optionalModuleIdParamSchema = schema.object({
+  moduleId: schema.maybe(schema.string()),
+});
 
-export const moduleIdParamSchema = getModuleIdParamSchema(false);
+export const moduleIdParamSchema = schema.object({ moduleId: schema.string() });
```

Comment:
This should have a doc comment added, as for the removed code above.

Reply:
added in bef6949